apps/cookbook/src/app/app-routing.module.ts

```diff
@@ -4,7 +4,12 @@ import { RouterModule } from '@angular/router';
 import { routes } from './app.routes';
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
+  imports: [
+    RouterModule.forRoot(routes, {
+      scrollOffset: [0, 0],
```

Comment:
Is this needed? 🤔

Reply:
Are you referring to `scrollOffset`?

Reply:
Yes

Reply:
I initially tought it was - but just tested without and it is not. I've removed it in a new commit 😄